src/cli/deploy.ts

```diff
@@ -45,7 +67,12 @@ export async function deploy(opts: { prefix?: string, customDomain?: string }):
 
     console.log("Uploading game...");
 
-    const archivedFile = await packageDir(PUBLISH_DIR);
+    let archivedFile;
+    try {
+        archivedFile = await packageDir(PUBLISH_DIR);
```

Comment:
is there any way to report upload progress?

Reply:
Yeah, I'm now reporting just on the archiving though. The can be in the scope of the full progress report (including unpacker and such)